ivy/array/sorting.py

```diff
@@ -17,7 +17,24 @@ def argsort(
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
         return ivy.argsort(self._data, axis, descending, stable, out=out)
+        """
```

Comment:
The docstring should be positioned above the return statement and below the function signature.